src/Navigation.ts

```diff
@@ -8,11 +8,13 @@ export default async ({ dynamicNav, currentNamespace }) => {
       const { extensions } = (await (await fetch(`/api/plugins/${pluginName}/plugin-manifest.json`))?.json()) || {};
       return extensions
         .filter(({ type, properties }) => type.includes('console.navigation') && properties.section === navSection)
-        .map((extension) => ({
-          appId,
-          href: `/${currentNamespace}/${navSection}${extension.properties.href}`,
-          title: extension.properties.name,
-        }));
+        .map((extension) => {
+          return {
+            appId,
+            href: `/${currentNamespace}/${navSection}${extension.properties.href}`,
+            title: extension.properties.name,
+          };
+        });
```

Comment:
Wanna undo just for a nicer diff?

Reply:
I'm fan of direct returns, whenever possible, it just feels more correct. Like whenever I see function body with just return it feels like something was left behind, some calculation was done there which is no longer needed.

Reply:
That's fair... but the `.filter` right above it is doing the same thing. Either way, I'm good... just figured it was debug stuff and left around 🙂

Reply:
That's a good point with the filter! I'll update it so it doesn't have the `return` statement.